### PR TITLE
Added a dark mode that works with OS preference

### PR DESCRIPTION
**Pull request by ardaa**
Added a dark mode using JS, HTML, and pure CSS. Works then prefers-color-scheme is 'dark'. Some package cleanup may be necessary as I installed some modules and wasn't able to purge them with my limited Golang knowledge. Also fixed an error typo.
TODO:
Add a toggle button that toggles between dark and light mode and bypassing OS preference

### CODE REVIEW

**Comment by boratanrikulu** [comment]
Hi,
Thank you for your contribution!   
I believe toggle button is a must for this feature. We should add it too.  
Can we improve dark theme style? It may be seem much better.

PR:
Screenshot_2021-01-18_23-06-43

Auto dark-reader theme is a good example:
Screenshot_2021-01-18_23-20-01

File: go.mod [hunk 1]
```diff
@@ -14,4 +14,6 @@ require (
 	github.com/joho/godotenv v1.3.0
 	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
 	google.golang.org/appengine v1.6.5 // indirect
+	gopkg.in/ini.v1 v1.62.0 // indirect
```
Comment (boratanrikulu) [inline go.mod]: I think these lines are not needed.

File: go.mod.save [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	github.com/magiconair/properties v1.8.4 // indirect
+	github.com/mitchellh/mapstructure v1.4.1 // indirect
+	github.com/pelletier/go-toml v1.8.1 // indirect
+	github.com/shopspring/decimal v1.2.0 // indirect
+	github.com/spf13/afero v1.5.1 // indirect
+	github.com/spf13/cast v1.3.1 // indirect
+	github.com/spf13/cobra v1.1.1 // indirect
+	github.com/spf13/jwalterweatherman v1.1.0 // indirect
+
+:q
```
Comment (boratanrikulu) [inline go.mod.save]: This file must be deleted.

File: go.sum [hunk 3]
```diff
@@ -1,7 +1,25 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```
Comment (boratanrikulu) [inline go.sum]: There's no need to change go.sum.

Comment (boratanrikulu) [reply go.sum]: Most of these packages are not used at Lirik.app

**Comment by ardaa** [comment]
Cool. Changing the color scheme to look like the image above. Sadly, I wasn't able to toggle the colors without using a CSS preprocessor or a bad JS hack. I really don't know Go so leaving that to other coders.

**Review by boratanrikulu — CHANGES_REQUESTED** [review]
Some changes are needed.

**Comment by boratanrikulu** [comment]
> Cool. Changing the color scheme to look like the image above. Sadly, I wasn't able to toggle the colors without using a CSS preprocessor or a bad JS hack. I really don't know Go so leaving that to other coders.

That's okay. We can open another issue for toggle button.  
For now, I will wait an update from you for css improvements and deleting unneeded commits(like changing go.mod or go.sum files).

**Comment by boratanrikulu** [comment]
Thank you again.  
I'll add an documentation to how to set-up the application on a local device.  
It's great to see you figure out that yourself.

**Comment by ardaa** [comment]
> Thank you again.
> I'll add an documentation to how to set-up the application on a local device.
> It's great to see you figure out that yourself.

I can also write the documentation on the setup process if you want.

**Comment by ardaa** [comment]
Screen Shot 2021-01-18 at 11 51 54 PM
Looks like this right now.

**Comment by boratanrikulu** [comment]
> Screen Shot 2021-01-18 at 11 51 54 PM
> Looks like this right now.

Hi,  
Seems much better! Could you do set tab titles to white and set `<hr>` style just like on the screenshot that sent by me?

**Comment by boratanrikulu** [comment]
> > Thank you again.
> > I'll add an documentation to how to set-up the application on a local device.
> > It's great to see you figure out that yourself.
> 
> I can also write the documentation on the setup process if you want.

Handling documentation by me much be better, thank you for offer.

**Comment by aykutkardas** [comment]
I think it has a lot of change. You can keep this simpler. For example using css variables. @ardaa 

```css
:root {
  --background: white;
  --foreground: black;
}

[data-theme='dark'] {
  --background: black;
  --foreground: white;
}
```

**Comment by boratanrikulu** [comment]
Kendimce bir koyu görünüm ekledim, bu pr'ı kapatıyorum
desteklerin için teşekkür ederim @ardaa

**Pull request closed without merging**